client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -68,6 +76,43 @@ class DebugTab extends Component {
 							</FormToggle>
 						</FormFieldset>
 						<div className="wp-super-cache__debug-fieldsets">
+							{ wp_cache_debug_log &&
+								<table>
+									<tr>
+										<td>
+											{ wp_super_cache_debug
+												? translate( 'Currently logging to:' )
+												: translate( 'Last logged to:' )
+											}
+										</td>
+										<td>
+											<ExternalLink
+												href={ cache_path_url + wp_cache_debug_log }
+												target="_blank">
+												{ cache_path + wp_cache_debug_log }
+											</ExternalLink>
+										</td>
+										<td rowSpan="2">
+											<Button
+												compact
+												disabled={ isRequesting || isSaving }
+												onClick={ this.deleteLog }
+												scary
+												value="1">
+												{ translate( 'Delete' ) }
+											</Button>
+										</td>
+									</tr>
+									<tr>
+										<td>
+											{ translate( 'Username and Password:' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **50** times:
`translate( 'Your username and password' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).